Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransporter.cs

```diff
@@ -48,13 +53,14 @@ internal FileTransporter(ISFTPLogger logger, BatchContext context, Guid instance
     /// <summary>
     /// Executes file transfers.
     /// </summary>
-    /// <param name="cancellationToken"></param>
     /// <returns></returns>
     /// <exception cref="DirectoryNotFoundException"></exception>
     /// <exception cref="FileNotFoundException"></exception>
     /// <exception cref="ArgumentException"></exception>
-    public FileTransferResult Run(CancellationToken cancellationToken)
+    public FileTransferResult Run()
     {
+        _logger.NotifyInformation(_batchContext, $"Connecting to {_batchContext.Connection.Address}:{_batchContext.Connection.Port} using SFTP.");
+
         var userResultMessage = "";
         try
         {

```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/29adcb25923403bc4ec0ccd337ffa6d50f72fd37/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransporter.cs#L51
This property is not used, so it can be removed.